content/en/docs/tasks/observability/distributed-tracing/jaeger/index.md

```diff
@@ -20,16 +20,53 @@ To learn how Istio handles tracing, visit this task's [overview](../overview/).
 
 1.  Follow the [Jaeger installation](/docs/ops/integrations/jaeger/#installation) documentation to deploy Jaeger into your cluster.
 
-1.  When you enable tracing, you can set the sampling rate that Istio uses for tracing.
-    Use the `meshConfig.defaultConfig.tracing.sampling` option during installation to
-    [set the sampling rate](/docs/tasks/observability/distributed-tracing/mesh-and-proxy-config/#customizing-trace-sampling).
-    The default sampling rate is 1%.
-
 1.  Deploy the [Bookinfo](/docs/examples/bookinfo/#deploying-the-application) sample application.
 
+## Configure Istio for distributed tracing
+
+### Configure an extension provider
+
+Install Istio with an [extension provider](/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig-ExtensionProvider) referring to the Jaeger collector service:
+
+{{< text bash >}}
+$ cat <<EOF > ./tracing.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
```

Comment:
What are your thoughts on providing on examples for the helm flow of installing Istio? For example, editing the configmap instead of IstioOperator?

Reply:
Agreed!